src/styles/ExpandableRow.ts

```diff
@@ -6,14 +6,12 @@ export const ExpandableRowStyles = () => {
   const expandedRowCSS = stylesheet({
     container: {
       width: '100%',
-      display: 'none',
       flexDirection: 'row',
       flexWrap: 'wrap',
       cursor: 'pointer',
       transition: 'border-radius 0.4s ease-in-out',
       $nest: {
         '&.content-row': {
-          display: 'none',
           cursor: 'default',
           $nest: {
             '&.content-row--expanded': {
```

Comment:
now you can merge the values for `.content-row--expanded` with `.content-row` above, and remove the rule `.content-row--expanded`. Indeed when displayed it will always be in the expanded state now.

Reply:
Yes, I intended to do that. Thanks for catching